README.md

```diff
@@ -14,11 +14,13 @@ A simple databaseless polling app for Slack
 
 ## How to Use
 ```
-/inorout Question
+/inorout
+Question
```

Comment:
This is incorrect, it is still 
```
/inorout question
Option
Option
```
unless it is multiple and/or anonymous then it's
```
/inorout [multiple] [anonymous]
question
opt1
opt2
.
.
```

Reply:
Okay, the following also works:
```
/inorout
question
opt1
opt2
.
.
```

Reply:
Maybe for consistency we just say it's 
```
/inorout
question
opt1
opt2
.
.
.
```
an then you add the options if you want.

Reply:
Sounds good, I'll leave the help docs as is.